lib/matplotlib/text.py

```diff
@@ -882,8 +882,9 @@ def get_window_extent(self, renderer=None, dpi=None):
             A renderer is needed to compute the bounding box.  If the artist
             has already been drawn, the renderer is cached; thus, it is only
             necessary to pass this argument when calling `get_window_extent`
-            before the first `draw`.  In practice, it is usually easier to
-            trigger a draw first (e.g. by saving the figure).
+            before the first draw.  In practice, it is usually easier to
```

Comment:
The `draw` link would go to [Text.draw](https://matplotlib.org/devdocs/api/text_api.html#matplotlib.text.Text.draw) which is more or less an internal API and not helpful here (at worst the user might try and call that). It's better to not link to anything here.